OverlayPlugin.Core/NetworkProcessors/LineEventStartPlayFinish.cs

```diff
@@ -0,0 +1,436 @@
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay64 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 64;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay128 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 128;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay255 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 255;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        public unsafe struct EventStart : IPacketStruct
+        {
+            uint actorID;
+            uint unknown1;
+            uint eventID;
+            byte param1;
+            byte param2;
+            ushort padding;
+            uint param3;
+            uint padding1;
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                return $"S|{actorID:X8}|{unknown1:X8}|{eventID:X8}|{param1:X2}|{param2:X2}|{param3:X8}";
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        public unsafe struct EventFinish : IPacketStruct
+        {
+            uint eventID;
+            byte param1;
+            byte param2;
+            ushort padding;
+            uint param3;
+            uint padding1;
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                return $"F|{eventID:X8}|{param1:X2}|{param2:X2}|{param3:X8}";
```

Comment:
Same comments as `EventStart` re: missing fields and `:X` formatting.  Kind of odd too that `actorID` isn't in the packet - perhaps we could just zero-fill that?